### DISCUSSION TITLE
Shouldn't Sidekiq also have a Docker depends_on ElasticSearch when enabled?

### DISCUSSION THREAD

**Discussion by hugogameiro** [mastodon/mastodon#18246] [Q&A]
When using Docker with ElasticSearch it is suggested that `depends_on: - es` is enabled on the Web container https://github.com/mastodon/mastodon/blob/main/docker-compose.yml#L59-L62

Just wondering why Sidekiq doesn't have the `depends_on: - es` since posts with mentions, that also need to go into the ElasticSearch database, that could be remote and federated via Sidekiq?

**Comment by ClearlyClaire** [comment]
Seems to me that `sidekiq` would `depend_on` `es` too in that case, yeah
